sal/class.json-api-site-base.php

```diff
@@ -427,6 +427,7 @@ function get_capabilities() {
 			'upload_files'        => current_user_can( 'upload_files' ),
 			'delete_users'        => current_user_can( 'delete_users' ),
 			'remove_users'        => current_user_can( 'remove_users' ),
+			'owns_site'           => wpcom_get_blog_owner() === (int) get_current_user_id(),
```

Comment:
Nice solution @Aurorum ❤️ I think our future selves will be glad we did this and didn't rely on the WordAds flag.

One suggestion - I had a look at `wpcom_get_blog_owner()` and in some circumstances it looks like figuring our the owner can be quite involved and involves several database queries. Shall we store the result in a variable rather than calling the function twice?

Reply:
Yep, that makes sense! I'm not sure how to test this though so fingers crossed that I haven't messed this up either :)